cmd/ncproxy/server_test.go

```diff
@@ -197,22 +198,46 @@ func TestAddNIC_HCN(t *testing.T) {
 			endpointName:  "",
 			errorExpected: true,
 		},
+		{
+			name:         "AddNIC returns no error with iov policy set",
+			containerID:  containerID,
+			nicID:        testNICID,
+			endpointName: "",
+			iovPolicySettings: &ncproxygrpc.IovEndpointPolicySetting{
+				IovOffloadWeight: 100,
+			},
+			errorExpected: true,
+		},
 	}
 
 	for _, test := range tests {
-		t.Run(test.name, func(_ *testing.T) {
+		t.Run(test.name, func(subtest *testing.T) {
+			endpointSettings := &ncproxygrpc.HcnEndpointSettings{}
+			if test.iovPolicySettings != nil {
+				if osversion.Build() < osversion.V21H1 {
```

Comment:
You'll need to import github.com/Microsoft/hcsshim/test/functional/manifest for osversion to work iirc. Or include a manifest/.syso in the ncproxy directory like the shim which we should probably be doing.